tests/integration/ceph_bucket_notification_test.go

```diff
@@ -103,6 +114,41 @@ func testBucketNotifications(s suite.Suite, helper *clients.TestClient, k8sh *ut
 		logger.Info("OBC, Secret and ConfigMap created")
 	})
 
+	t.Run("perform s3 operations and check for notifications", func(t *testing.T) {
+		var s3client *rgw.S3Agent
+		s3endpoint, _ := helper.ObjectClient.GetEndPointUrl(namespace, storeName)
+		s3AccessKey, _ := helper.BucketClient.GetAccessKey(obcName)
+		s3SecretKey, _ := helper.BucketClient.GetSecretKey(obcName)
+		if objectStore.Spec.IsTLSEnabled() {
+			s3client, err = rgw.NewInsecureS3Agent(s3AccessKey, s3SecretKey, s3endpoint, region, true)
+		} else {
+			s3client, err = rgw.NewS3Agent(s3AccessKey, s3SecretKey, s3endpoint, region, true, nil)
+		}
+
+		assert.Nil(t, err)
+		logger.Infof("endpoint (%s) Accesskey (%s) secret (%s)", s3endpoint, s3AccessKey, s3SecretKey)
+
+		t.Run("put object", func(t *testing.T) {
```

Comment:
I guess negative cases are missing atm, from the docs I guess we can check notification with put and delete easily. For others, we may need to add new s3 helpers. I can add negative cases using these two as well. Please share your thoughts as well.

Reply:
a simple negative case would be to define only the "create" event, and verify there is no notification after an object deletion.
similarly, you can define only a "delete" event, and verify there is a notification only after the deletion

Reply:
Thats what in mind as well